src/components/activities.js

```diff
@@ -216,7 +231,13 @@ export class ActivityIframePort {
    * @template T
    */
   on(message, callback) {
-    const label = getLabel(message);
+    let label = null;
+    try {
+      label = getLabel(message);
+    } catch (ex) {
+      // Thrown is message is not a proto object
```

Comment:
I'm having trouble parsing this sentence. Could you try rewriting for a dummy like me? :)